README.md

```diff
@@ -61,10 +61,15 @@ For more details, including how to join and recordings of previous syncs, see th
 
 ## Contributing to rkt
 
-rkt is an open source project under the Apache 2.0 [license](LICENSE), and contributions are gladly welcomed!
+rkt is an open source project and contributions are gladly welcomed!
 See the [Hacking Guide](Documentation/hacking.md) for more information on how to build and work on rkt.
 See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
 
+## Licensing
+
+Unless otherwise noted, all code in the rkt repository is licensed under the [Apache 2.0 license](LICENSE).
+Some portions of the codebase are derived from other projects under difference licenses; the appropriate information can be found in the header of those source files, as applicable.
```

Comment:
fixed

On Mon, Feb 22, 2016 at 1:33 PM, Krzesimir Nowak <notifications@github.com>
wrote:

> In README.md
> <https://github.com/coreos/rkt/pull/2176#discussion_r53618394>:
>
> >  See the [Hacking Guide](Documentation/hacking.md) for more information on how to build and work on rkt.
> >  See [CONTRIBUTING](CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
> >
> > +## Licensing
> > +
> > +Unless otherwise noted, all code in the rkt repository is licensed under the [Apache 2.0 license](LICENSE).
> > +Some portions of the codebase are derived from other projects under difference licenses; the appropriate information can be found in the header of those source files, as applicable.
>
> s/difference/different/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2176/files#r53618394>.
>